tensorboard/webapp/metrics/views/card_renderer/image_card_component.ts

```diff
@@ -70,12 +70,8 @@ export class ImageCardComponent {
     return `contrast(${contrastPercent}%) brightness(${brightnessScale})`;
   }
 
-  onSliderInput($event: any) {
-    // Angular Material Slider's MatLegacySliderChange has a loose `number | null`
-    // type for 'value'. However, it's actual implementation can only emit a
-    // `number` on input events.
-    // https://github.com/angular/components/blob/master/src/material/slider/slider.ts
-    this.stepIndexChange.emit($event.value as number);
+  onSliderInput(value: number) {
+    this.stepIndexChange.emit(value as number);
```

Comment:
I think you can remove `as number` now, basd on the comment you deleted.

```
this.stepIndexChange.emit(value);
```

Reply:
Good catch. Removed.